ff2mpv.py

```diff
@@ -1,22 +1,29 @@
 #!/usr/bin/env python3
 
-import sys
-import struct
 import json
-from subprocess import Popen, DEVNULL
+import platform
+import struct
+import sys
+from subprocess import Popen, CREATE_BREAKAWAY_FROM_JOB
 
 
 def main():
     message = get_message()
     url = message.get("url")
 
-    mpv_args = ["--no-terminal"]
+    args = ["mpv", "--no-terminal", "--", url]
+
+    kwargs = {}
+    # https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging#Closing_the_native_app
+    if platform.system() == "Windows":
+        kwargs["creationflags"] = CREATE_BREAKAWAY_FROM_JOB
+
+    Popen(args, **kwargs)
 
-    args = ["mpv", *mpv_args, "--", url]
-    Popen(args, stdin=DEVNULL, stdout=DEVNULL, stderr=DEVNULL)
     # Need to respond something to avoid "Error: An unexpected error occurred"
     # in Browser Console.
-    send_message("ok")
+    if url:
```

Comment:
Let's go back to the prior message. This has never been an issue with the Ruby client, so I still this there's something else wrong here.

Reply:
(But that's not your fault or problem; I just want to keep this change minimal).